README.md

```diff
@@ -1,7 +1,23 @@
-# Node.js skeleton repo for Google STEP Intern Program
+# Donatia
+Researching nonprofit organization to donate to can be tedious. There are roughly 1.5 million nonprofit organizations in the United States, each with their own mission, acceptable donation items, and methods for donating. With so much variety, it can be difficult to find the best organization that seeks your donation item and whose donation process is convient for you.
 
-## Get started 
-You need to have an active Node.js installation on your laptop before startup this skeleton Node.js server. We recommend to install `nvm` [(Node Version Manager)](https://github.com/nvm-sh/nvm) to manage your local Node.js builds.
+Donatia simplifies this process by providing an online platform to let donors filter through find organizations easily. By creating a streamlined way to discover nonprofit organizations, we hope to reduce waste and increase donations.
+
+![image](https://user-images.githubusercontent.com/10712922/87168036-e707f800-c29b-11ea-8d43-698253bdef57.png)
+
+![image](https://user-images.githubusercontent.com/10712922/87168160-1159b580-c29c-11ea-8e95-333a9945adc6.png)
+
+![image](https://user-images.githubusercontent.com/10712922/87168250-32220b00-c29c-11ea-81bc-e63f0a46c705.png)
+
+## Built With
+- [Handlebars](https://handlebarsjs.com/)
+- [Node.js](https://nodejs.org/en/)
+- [Express.js](https://expressjs.com/)
+- [Firebase](https://firebase.google.com/)
```

Comment:
We are just using the Google Cloud version of Firestore so not technically using Firebase which has Firestore and other APIs
Cloud Firestore (https://cloud.google.com/firestore)